src/ui/App.tsx

```diff
@@ -1,13 +1,18 @@
-import NotebookFooter from "./NotebookFooter";
-import NotebookHeader from "./NotebookHeader";
-import NotebookView from "./NotebookView";
+import { Route, Switch } from "wouter";
+import Home from "~/pages/Home";
+import Notebook from "~/pages/Notebook";
 
 export default function App() {
   return (
     <main className="absolute inset-0 bg-white">
-      <NotebookHeader kernelName="Local Kernel (Python 3.11.7)" />
-      <NotebookView />
-      <NotebookFooter />
+      <Switch>
```

Comment:
Do you need a top-level `<Router>` provider? I was looking at the wouter docs and wasn't sure whether we could use a hash router or just a path-based one.

It seems like tauri places the webview URL in prod as `app://index.html` https://docs.rs/tauri-utils/2.1.1/src/tauri_utils/config.rs.html#108

so a normal path might not work, maybe needs to be a hash-based route? Not sure